config.py

```diff
@@ -26,13 +26,17 @@ class Config:
     MultiworldRomBase = os.environ.get("MultiworldRomBase")
     MultiworldHostBase = os.environ.get("MultiworldHostBase")
 
-    gsheet_api_oauth = json.loads(os.environ.get("gsheet_api_oauth"), strict=False)
+    gsheet_api_oauth = os.environ.get("gsheet_api_oauth")
+    if gsheet_api_oauth is not None:
+        json.loads(gsheet_api_oauth, strict=False)
 
     SB_TWITCH_TOKEN = os.environ.get("SB_TWITCH_TOKEN")
     SB_TWITCH_CLIENT_ID = os.environ.get("SB_TWITCH_CLIENT_ID")
     SB_TWITCH_NICK = os.environ.get("SB_TWITCH_NICK")
     SB_TWITCH_PREFIX = os.environ.get("SB_TWITCH_PREFIX")
-    SB_TWITCH_CHANNELS = os.environ.get("SB_TWITCH_CHANNELS").split(',')
+    SB_TWITCH_CHANNELS = os.environ.get("SB_TWITCH_CHANNELS")
+    if SB_TWITCH_CHANNELS is not None:
+        SB_TWITCH_CHANNELS = SB_TWITCH_CHANNELS.split(',')

```

Comment:
Originally when running python sahasrahbot.py locally, it would fail in this file and also would not "compile" my changes.  Thus I added a couple of sanity checks here, and now I can at least verify python can parse my choozo logic.